src/components/library-app-2/book-form.js

```diff
@@ -1,218 +1,256 @@
-// import React, {useState} from 'react';
-// import CheckBox from '@react-native-community/checkbox';
-// import {
-//   StatusBar,
-//   StyleSheet,
-//   KeyboardAvoidingView,
-//   ScrollView,
-//   Pressable,
-//   Text,
-//   TextInput,
-//   View,
-// } from 'react-native';
-// import FontAwesome from 'react-native-vector-icons/FontAwesome';
-// import SelectDropdown from 'react-native-select-dropdown';
+import React, {useState} from 'react';
```

Comment:
UI Wise:
Submit button stuck to bottom lift it. It should be completely visible

Reply:
done